src/utils/transformations.js

```diff
@@ -109,7 +109,13 @@ function getParsedInputFromTransaction(transaction) {
   }
 }
 
-function getTransactionDescription(transaction, tokensByAddress, getReadableOrder, getReadableSwapOrder) {
+function getTransactionDescription(
```

Comment:
Can you just add a different description for erc721s? ({tokenSymbol} #{tokenParam}) Since we have to approve per nft item. i.e. Approve CK #2 for trade